src/main/java/com/arpnetworking/metrics/impl/ApacheHttpSink.java

```diff
@@ -50,98 +54,116 @@
  * Http sink using the Protobuf format for metrics and the Apache HTTP library.
  *
  * @author Brandon Arp (brandon dot arp at inscopemetrics dot com)
+ * @author Ville Koskela (ville dot koskela at inscopemetrics dot com)
  */
 public final class ApacheHttpSink implements Sink {
+
+    /**
+     * {@inheritDoc}
```

Comment:
We've discussed this. While I completely agree we should do away with it the change needs to be backfilled. When that's done I'll stop adding it. I would prefer we stick to our conventions or then actually change them. Living in between is a lack of convention.